giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/users/UsersManager.java

```diff
@@ -138,13 +163,19 @@ public void revokeUserAccount(String username) {
     public void addRoleToUser(String username, GilesRole role) {
         User user = findUser(username);
         user.getRoles().add(role.name());
+        if(role == GilesRole.ROLE_ADMIN) {
```

Comment:
why are you adding this? is there are reason you're not using the roles to decide if a user is an admin?